cime/config/e3sm/config_grids.xml

```diff
@@ -754,6 +754,26 @@
       <mask>oQU240</mask>
     </model_grid>
 
+    <model_grid alias="ne4pg3_ne4pg3" compset="(DOCN|XOCN|SOCN|AQP1)">
+      <grid name="atm">ne4np4.pg3</grid>
```

Comment:
Any particular motivation for using the "dot" to separate dynamics from physics grids here?

Reply:
That's how Peter Lauritzen did it in CAM-SE, so I followed his example.